This says "Freebsd compilation fixes", should this close #18249?

Because I still get netlink errors compiling this fork on FreeBSD:

```
$ git remote -v
origin	https://github.com/mrjana/docker.git (fetch)
origin	https://github.com/mrjana/docker.git (push)
$ git status
On branch mh
Your branch is up-to-date with 'origin/mh'.
nothing to commit, working directory clean
$ git log -1
ESC[33mcommit 5b450e97387e9f87550b5cb58b3d4efa85394494ESC[m
Author: Jana Radhakrishnan <mrjana@docker.com>
Date:   Fri Dec 18 11:03:57 2015 -0800

    libnetwork vendoring
    
    Vendoring libnetwork @ df3f8a10796e2173732aeafa41af45f970ced9f6
    
    Brings in:
        * Support for overlay network driver in 3.10+ kernels
        * Freebsd compilation fixes
        * Remove .dockerinit dependency
        * IPAM driver capability support
    
    Signed-off-by: Jana Radhakrishnan <mrjana@docker.com>
$ export AUTO_GOPATH=1 && ./hack/make.sh binary
# WARNING! I don't seem to be running in the Docker container.
# The result of this command might be an incorrect build, and will not be
#   officially supported.
#
# Try this instead: make all
#

date: illegal option -- -
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ... 
            [-f fmt date | [[[[[cc]yy]mm]dd]HH]MM[.ss]] [+format]
bundles/1.10.0-dev already exists. Removing.

---> Making bundle: binary (in bundles/1.10.0-dev/binary)
Building: bundles/1.10.0-dev/binary/docker-1.10.0-dev
vendor/src/github.com/vishvananda/netlink/filter.go:6:2: no buildable Go source files in /home/clint/docker/vendor/src/github.com/vishvananda/netlink/nl
```

--- a/docs/userguide/networking/get-started-overlay.md
+++ b/docs/userguide/networking/get-started-overlay.md
@@ -16,7 +16,6 @@ network. Docker Engine supports multi-host networking out-of-the-box through the
 `overlay` network driver.  Unlike `bridge` networks, overlay networks require
 some pre-existing conditions before you can create one. These conditions are:
 
-* A host with a 3.16 kernel version or higher.
 * Access to a key-value store. Docker supports Consul, Etcd, and ZooKeeper (Distributed store) key-value stores.
 * A cluster of hosts with connectivity to the key-value store.
 * A properly configured Engine `daemon` on each host in the cluster.
@@ -46,7 +45,7 @@ key-value stores. This example uses Consul.
 
 1. Log into a system prepared with the prerequisite Docker Engine, Docker Machine, and VirtualBox software.
 
-2. Provision a VirtualBox machine called `mh-keystore`.  
+2. Provision a VirtualBox machine called `mh-keystore`.
 
 		$ docker-machine create -d virtualbox mh-keystore
 

--- a/hack/vendor.sh
+++ b/hack/vendor.sh
@@ -23,7 +23,7 @@ clone git golang.org/x/net 47990a1ba55743e6ef1affd3a14e5bac8553615d https://gith
 clone git github.com/docker/go-units v0.2.0
 
 #get libnetwork packages
-clone git github.com/docker/libnetwork bbd6e6d8ca1e7c9b42f6f53277b0bde72847ff90
+clone git github.com/docker/libnetwork 9f0563ea8f430d8828553aac97161cbff4056436
 clone git github.com/armon/go-metrics eb0af217e5e9747e41dd5303755356b62d28e3ec
 clone git github.com/hashicorp/go-msgpack 71c2886f5a673a35f909803f38ece5810165097b
 clone git github.com/hashicorp/memberlist 9a1e242e454d2443df330bdd51a436d5a9058fc4

